### ISSUE TITLE
yarac sporadically fails when trying to specify namespace for files

### ISSUE THREAD

**Issue by redbaron4** [VirusTotal/yara#1202]
We use `yarac` to compile all yara files into a single file. Since update to yara-3.11.0 (through EPEL), yarac command has been failing intermittently with "file not found" as seen in below screen paste

```
[root@XX rules-master]# ll
total 156
drwxr-xr-x. 2 root root    34 Jan 19 03:55 antidebug_antivm
-rw-r--r--. 1 root root    94 Jan 23 13:16 antidebug_antivm_index.yar
drwxr-xr-x. 2 root root    30 Jan 19 03:55 capabilities
-rw-r--r--. 1 root root    86 Jan 23 13:16 capabilities_index.yar
drwxr-xr-x. 2 root root    35 Jan 19 03:55 crypto
-rw-r--r--. 1 root root    85 Jan 23 13:16 crypto_index.yar
drwxr-xr-x. 2 root root  4096 Jan 19 03:55 cve_rules
-rw-r--r--. 1 root root   606 Jan 23 13:16 cve_rules_index.yar
drwxr-xr-x. 2 root root  4096 Jan 23 13:16 deprecated
drwxr-xr-x. 3 root root  4096 Jan 19 03:55 email
-rw-r--r--. 1 root root   281 Jan 23 13:16 email_index.yar
drwxr-xr-x. 2 root root   254 Jan 19 03:55 exploit_kits
-rw-r--r--. 1 root root   490 Jan 23 13:16 exploit_kits_index.yar
-rwxr-xr-x. 1 root root  2096 Jan 19 03:55 index_gen.sh
-rw-r--r--. 1 root root 18436 Jan 23 13:16 index_w_mobile.yar
-rw-r--r--. 1 root root 18436 Jan 23 13:16 index.yar
-rw-r--r--. 1 root root 18047 Jan 19 03:55 LICENSE
drwxr-xr-x. 2 root root  4096 Jan 19 03:55 maldocs
-rw-r--r--. 1 root root   904 Jan 23 13:16 maldocs_index.yar
drwxr-xr-x. 3 root root 12288 Jan 23 13:16 malware
-rw-r--r--. 1 root root 15631 Jan 23 13:16 malware_index.yar
drwxr-xr-x. 2 root root    22 Jan 19 03:55 mobile_malware
-rw-r--r--. 1 root root    44 Jan 23 13:16 mobile_malware_index.yar
drwxr-xr-x. 2 root root   144 Jan 19 03:55 packers
-rw-r--r--. 1 root root   247 Jan 23 13:16 packers_index.yar
-rw-r--r--. 1 root root  3768 Jan 19 03:55 README.md
drwxr-xr-x. 3 root root   176 Jan 19 03:55 utils
drwxr-xr-x. 2 root root   250 Jan 19 03:55 webshells
-rw-r--r--. 1 root root   408 Jan 23 13:16 webshells_index.yar
[root@XX rules-master]# yarac cve_rules:cve_rules_index.yar ../compiled.yarac
[root@XX rules-master]# yarac cve_rules:cve_rules_index.yar ../compiled.yarac
error: could not open file: cve_rules:cve_rules_index.yar
```

AFAIK this wasn't an issue in prior versions.

**Comment by plusvic** [comment]
The same command sometimes fails and sometimes succeed?

**Comment by plusvic** [comment]
Ok, the issue is easily reproducible. It was fixed in 5424f0fab3db92784d9b3bf337107446d81b3e7b

**Comment by redbaron4** [comment]
@plusvic Thanks for the quick fix. Is there a planned date for the next release?
